docs/src/wallet/ledger-live.md

```diff
@@ -0,0 +1,52 @@
+#Ledger Live and Ledger Nano S
+This document describes how to set up a
+[Ledger Nano S hardware wallet](https://shop.ledger.com/products/ledger-nano-s)
+with the [Ledger Live](https://www.ledger.com/ledger-live) software.
+
+**NOTE: While Solana tools are fully integrated with the Ledger Nano S device,
+and the Solana App can be installed on the Nano S using Ledger Live, adding and
+managing wallet accounts currently requires use of our command-line tools.
+Integration with Ledger Live to use Solana wallet accounts on Ledger Live
+will be available in the future.**
+
+Users may use a Ledger Nano S with the Solana command line tools.
+Instructions for how to do that are found [here](../remote-wallet/ledger.md).
+
+##Set up a Ledger Nano S
+ - Order a Nano S from Ledger [here](https://shop.ledger.com/products/ledger-nano-s)
+ - Follow the instructions for device setup included in the package,
+ or [Ledger's Start page](https://www.ledger.com/start/)
+
+##Install Ledger Live
+ - Install [Ledger Live desktop software](https://www.ledger.com/ledger-live/),
+ or
+ - Install the Ledger Live app for [iOS](https://apps.apple.com/app/id1361671700)
+ or [Android](https://play.google.com/store/apps/details?id=com.ledger.live).
+   - Requires iOS 9.1 or later. Compatible with iPhone, iPad, and iPod touch.
+   - Requires Android 7.0 or later.
+ - Connect your Nano S to your device and follow the instructions
+
+##Install the Solana App on your Nano S
+ - Open Ledger Live
+ - Go to Manager in the app and find "Solana (SOL)" in the App Catalog and
+ click Install
+   - Make sure your device is plugged in via USB and is unlocked with its PIN
+ - You may be prompted on the Nano S to confirm the install of Solana App
+ - "Solana (SOL)" should now show as Installed in the Ledger Live Manager
+
+## Future Features (Not yet supported)
```

Comment:
It might pay to omit the unsupported features for now

Reply:
Yeah I thought about that, probably would reduce confusion.  At that point, this page just becomes a verbose duplicate of https://github.com/solana-labs/solana/blob/master/docs/src/remote-wallet/ledger.md#install-the-solana-app-on-ledger-nano-s

So should this whole page be cut out until we have address/send/receive on Ledger Live?  Maybe people will feel its misleading that they can do their familiar Nano workflow, then surprise!  You gotta use the CLI.

Reply:
Maybe merge them?  The mobile app bits are new AFAIK

Reply:
Should I just list without instructions, something like the below, or just strike the whole section?

Future features to be added:
 - Ability to view wallet address on Ledger Live
 - Ability to send and receive SOL tokens on Leger Live

Reply:
I'd probably just strike it for now, since that integration is mostly out of our hands

Reply:
Done.  Just added another link at the end pointing them to the Nano CLI page